xml/System.CommandLine.Builder/CommandLineBuilderExtensions.xml

```diff
@@ -0,0 +1,700 @@
+        <AssemblyVersion>2.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.CommandLine.Builder.CommandLineBuilder</ReturnType>
+      </ReturnValue>
+      <Parameters>
+        <Parameter Name="builder" Type="System.CommandLine.Builder.CommandLineBuilder" RefType="this" />
+        <Parameter Name="value" Type="System.Boolean" />
+      </Parameters>
+      <Docs>
+        <param name="builder">To be added.</param>
+        <param name="value">To be added.</param>
+        <summary>
+            Determines the behavior when parsing a double dash (<c>--</c>) in a command line.
+            </summary>
+        <returns>To be added.</returns>
+        <remarks>When set to <see langword="true" />, all tokens following <c>--</c> will be placed into the <see cref="P:System.CommandLine.Parsing.ParseResult.UnparsedTokens" /> collection. When set to <see langword="false" />, all tokens following <c>--</c> will be treated as command arguments, even if they match an existing option.</remarks>
+      </Docs>
+    </Member>
+    <Member MemberName="EnablePosixBundling">
+      <MemberSignature Language="C#" Value="public static System.CommandLine.Builder.CommandLineBuilder EnablePosixBundling (this System.CommandLine.Builder.CommandLineBuilder builder, bool value = true);" />
+      <MemberSignature Language="ILAsm" Value=".method public static hidebysig class System.CommandLine.Builder.CommandLineBuilder EnablePosixBundling(class System.CommandLine.Builder.CommandLineBuilder builder, bool value) cil managed" />
+      <MemberSignature Language="DocId" Value="M:System.CommandLine.Builder.CommandLineBuilderExtensions.EnablePosixBundling(System.CommandLine.Builder.CommandLineBuilder,System.Boolean)" />
+      <MemberSignature Language="VB.NET" Value="&lt;Extension()&gt;&#xA;Public Function EnablePosixBundling (builder As CommandLineBuilder, Optional value As Boolean = true) As CommandLineBuilder" />
+      <MemberSignature Language="F#" Value="static member EnablePosixBundling : System.CommandLine.Builder.CommandLineBuilder * bool -&gt; System.CommandLine.Builder.CommandLineBuilder" Usage="System.CommandLine.Builder.CommandLineBuilderExtensions.EnablePosixBundling (builder, value)" />
+      <MemberType>Method</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>System.CommandLine</AssemblyName>
+        <AssemblyVersion>2.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.CommandLine.Builder.CommandLineBuilder</ReturnType>
+      </ReturnValue>
+      <Parameters>
+        <Parameter Name="builder" Type="System.CommandLine.Builder.CommandLineBuilder" RefType="this" />
+        <Parameter Name="value" Type="System.Boolean" />
+      </Parameters>
+      <Docs>
+        <param name="builder">A command line builder.</param>
+        <param name="value">If set to <see langword="true" />, then POSIX bundles are parsed. ; otherwise, <see langword="false" />.</param>
```

Comment:
```suggestion
        <param name="value">
          <see langword="true" /> to parse POSIX bundles; otherwise, <see langword="false" />.</param>
```